## template_url_prepopulate_data.cc
# Adds DDG to the default search engines.
#
--- a/components/search_engines/template_url_prepopulate_data.cc
+++ b/components/search_engines/template_url_prepopulate_data.cc
@@ -64,6 +64,7 @@ constexpr EngineAndTier engines_default[
     {SearchEngineTier::kTopEngines, &google},
     {SearchEngineTier::kTopEngines, &bing},
     {SearchEngineTier::kTopEngines, &yahoo},
+    {SearchEngineTier::kTopEngines, &duckduckgo},
 };
 
 // Note, the below entries are sorted by country code, not the name in comment.
